src/Button.js

```diff
@@ -0,0 +1,6 @@
+// functional component
+import React from "react";
```

Comment:
Small hack, from `react` v17, you can omit this import and it won't be an error.